CMakeLists.txt

```diff
@@ -65,9 +77,17 @@ set(SERENITY_TEST_SOURCES
     tests/controllers/qos_controller_test.cpp
     tests/estimators/estimator_test.cpp
     tests/filters/ema_test.cpp
+    tests/observers/slack_resource_test.cpp
     tests/sources/json_source_test.cpp
 )
 
+set(LIBC_MOCK_SOURCES
+    tests/common/mocks/time.cpp
+)
+
+#libc mock library
```

Comment:
Artifact of other work :)

Wanted to mock libc function by ld_preload my SO :)